pages/cart.html

```diff
@@ -0,0 +1,118 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Cart-page</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css"
+    />
+    <link href="https://asthra-ui.netlify.app/style.css" rel="stylesheet">
+    <link href="/css/style.css" rel="stylesheet">
+    <link href="/css/cart.css" rel="stylesheet">
+    
+</head>
+<body>
+    <nav class = "nav-bar">
+        <span class ="nav-title">BakeHere</span>
+        <div class = "input-box-icon">
+            <label for = "input-search"></label>
+            <div class = "input-icon-container border-grey">
+                <button><i class="bi bi-search"></i></button>
+                <input type = "text" id ="input-search" placeholder="Search">
+            </div>
+        </div>
+        <div class = "btn-nav">
+            <button class="btn-login">
+              <a href = "/" >Login</a> 
+            </button>
+            <button class ="btn-icon">
+                <a href="/"><i class="bi bi-heart-fill"></i></a>
+            </button>
+            <button class ="btn-icon">
+                <a href="/pages/cart.html"><i class="bi bi-cart-fill"></i></a>
+            </button>
+        </div>
+    </nav>
+    <h1 class = "center-text">My Cart (2)</h1>
+
+    <div class = "cart-page-container">
+
+        <div class = "cart-card-container">
+            <div class="basic-card basic-card-horizontal border-radius-5px">
+                <div class = "img-section">
+                    <img class ="card-img card-flex-item border-radius-5px" src="/assets/heart-shape-mould.jpeg">
+                    <span class="badge-on-card">New</span>
+                </div>
+                <div class="card-body">
+                    <p>Trendy Elegant Women Tops & Tunics</p>
+                    <div class = "price-details">
+                        <span class = "new-price">&#x20B9 158</span>
+                        <span class = "old-price">&#x20B9 225</span>
+                        <span class = "offer">30% off</spam>
+                    </div>
+                        <span>Quantity: </span>
+                        <div class = "quantiy-section">
+                            <button>-</button>
+                            <input type = "number" value = "1"/>
+                            <button>+</button>
+                        </div>
+                    <button class="btn-addtocart border-radius-5px"></i>Move To Wishlist</button>
+                    <button class="btn-addtocart border-radius-5px btn-remove"></i>Remove From Cart</button>
+                </div>
+            </div>
+            <div class="basic-card basic-card-horizontal border-radius-5px">
+                <div class = "img-section">
+                    <img class ="card-img card-flex-item border-radius-5px" src="/assets/heart-shape-mould.jpeg">
+                    <span class="badge-on-card">New</span>
```

Comment:
Same here use alt for better accessibility.

Reply:
Thanks for the suggestion. I have made the changes.